docs/services/zwo_camera.rst

```diff
@@ -1,14 +1,74 @@
 ZWO Camera
 ==========
 
+This service operates a ZWO camera. There are currently four different types of ZWO cameras that have been tested and used with catkit2 so far:
+- ZWO ASI533MM (https://www.zwoastro.com/product/asi533mm-mc/)
+- ZWO ASI290MM (https://agenaastro.com/zwo-asi290mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI178MM (https://agenaastro.com/zwo-asi178mm-cmos-monochrome-astronomy-imaging-camera.html)
+- ZWO ASI1600MM (https://agenaastro.com/zwo-asi1600mm-p-cmos-monochrome-astronomy-imaging-camera-pro.html)
+
+For camera specs, see the website links above.
+
 Configuration
 -------------
 
+.. code-block:: YAML
+
+    camera1:
+        service_type: zwo_camera
+        simulated_service_type: camera_sim
+        requires_safety: false
+
+        device_name: ZWO ASI533MM
+        offset_x: 1038
+        offset_y: 1282
+        width: 192
+        height: 192
+        exposure_time: 1000
+        gain: 100
+
 Properties
 ----------
+``exposure_time``: Exposure time of the camera.
+
+``gain``: Gain of the camera.
+
+``brightness``: Brightness of the camera.
+
+``width``: The width of the camera.
+
+``height``: The height of the camera.
+
+``offset_x``: The x offset of the camera.
+
+``offset_y``: The y offset of the camera.
+
+``sensor_width``: The width of the sensor.
+
+``sensor_height``: The height of the sensor.
+
+``device_name``: The name of the camera.
 
 Commands
 --------
+``start_acquisition()``: This starts the acquisition of images from the camera.
+
+``end_acquisition()``: This ends the acquisition of images from the camera.
+
+# Additional commands below inherited from the Service Proxy class
+``take_calibrated_exposures(num_exposures, only_use_cache=True)``: This takes a number of calibrated exposures.
+
+``get_exposure_calibration_function(only_use_cache=True)``: This gets a function to calibrate individual exposures.
+
+``take_calibrated_image(num_exposures, upsample_factor=None, alignment_window=None, only_use_cache=True)``: This takes a calibrated 
+image which is background subtracted, exposure time corrected, nd flux calibrated and sub-frame aligned using phase cross correlation method.
+
+``take_dark(num_exposures)``: This takes a dark image for the current exposure time by moving the beam dump.
```

Comment:
Careful, this is again from the HiCAT proxy, so these need to be deleted. If you want to make a note about the generic `camera` proxy, then it might be best to do this in the general description up top in this doc.